.ci/prep.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -euo pipefail
+
+set -x
+export MYSQL_VERSION="${MYSQL_VERSION:-8.4}"
+export MYSQL_CERTS_DIR='/etc/mysql_certs'
+
+mkdir -p .ci/run .ci/certs
+SSLDIR=/etc/mysql_certs make tests-prep
+sudo cp test/ssl/ca.pem .ci/certs/
+sudo mv test/ssl/server-key.pem .ci/certs/
+sudo mv test/ssl/server-cert.pem .ci/certs/
+mv test/ssl/my-ssl.cnf .ci/
+sudo chmod 660 .ci/certs/*
```

Comment:
This is for running the tests using docker, not necessarily in a CI but also locally?

Why does this need sudo? Do the certs need to be owned by root? sudo may prompt for password so maybe it can be a problem to use it in the script? (In Github Actions, it's configured to not need a password I guess, but if we want to use this from some other place, such as running it locally.)

Reply:
added a comment to explain why sudo is needed here